support-models/src/main/scala/com/gu/support/workers/Products.scala

```diff
@@ -61,7 +63,7 @@ case class GuardianWeekly(
 object ProductType {
   import com.gu.support.encoding.CustomCodecs._
   implicit val decoderDigital: Decoder[DigitalPack] = deriveDecoder[DigitalPack]
-    .prepare(_.withFocus(_.mapObject(_.checkKeyExists("productOptions", Json.fromString("NoProductOptions")))))
+    .prepare(_.withFocus(_.mapObject(_.checkKeyExists("readerType", Json.fromString("Direct")))))
```

Comment:
I realise it's existing but what is this for? it looks like it's saying that we will refuse to decide non-Direct digitalpack, is that just a safety check until corporate and gift are released?

Reply:
It's to handle defaulting the `readerType` to `Direct` if it is not present in the Json. Circe doesn't respect default values in case classes so we need to do a bit more work to avoid having to add it in everywhere.

Reply:
ah I see so "check" is more like "ensure" rather than "assert", makes sense now